homework-submission/week2/app.js

```diff
@@ -0,0 +1,101 @@
+class TodoListItem extends React.Component {
+  render() {
+    const { id, description, deadline, done, ClickHandler } = this.props;
```

Comment:
no need to make ClickHandler in capital-case 😉

Reply:
done